src/opt.rs

```diff
@@ -56,6 +56,14 @@ pub(crate) struct AppConfig {
     #[structopt(long, group = "target", value_name = "NAME")]
     bench: Option<String>,
 
+    /// Test harness target to run
```

Comment:
Updated the code, will work on docs next. Just a note that technically it doesn't have to be an integration test. All tests generate their binaries. When you run `cargo test`, you'll see something like

```
[...]
Running unittests src/main.rs (target/debug/deps/cargo_instruments-3cabce0a041cfeed)
```

which means that you can do:
```
cargo instruments -t alloc --test cargo-instruments -- opt::tests::features
```

Reply:
okay that's great. In an ideal world then maybe you should be able to omit the target after `--test`, and we would run the "default test target", if such a thing exists, and if we can figure out how to get it from cargo? This might involve some splunking in the cargo crate internals to figure out what they do when you run `cargo test`.